statements.php

```diff
@@ -0,0 +1,115 @@
+<!--
+This file contains functions that returns sql queries for GET requests.
+This file returned data based on data specified by API documentation.
+-->
```

Comment:
Why is this called `statements.php`? Use future tense: `returns` instead of `returns`. Does the *file* return data or does the file contain helper functions which return such data?